src/main/java/io/github/thebusybiscuit/slimefun4/core/attributes/EnergyNetProvider.java

```diff
@@ -48,6 +49,19 @@ default EnergyNetComponentType getEnergyComponentType() {
      */
     int getGeneratedOutput(@Nonnull Location l, @Nonnull Config data);
 
+    /**
+     * The method return how much this {@link EnergyNetProvider} is expected to provide to the {@link EnergyNet}
+     * and is not to be confused with {@link #getGeneratedOutput(Location, Config)}. This method will not
+     * consume any fuel or modify the provider in any way.
```

Comment:
English here can be improved.
```suggestion
     * This method will return how much this {@link EnergyNetProvider} is expected to provide to the {@link EnergyNet}.
     * It is not to be confused with {@link #getGeneratedOutput(Location, Config)}.
     * This method <b>should not</b> consume any fuel or modify the provider in any way.
```